src/components/HeroCard/herocard.css

```diff
@@ -0,0 +1,78 @@
+.hero {
+  position: relative;
+  display: flex;
+  height: 100vh;
+  width: 100%;
+  padding: 16px 16px;
+}
```

Comment:
trên điện thoại, viewport cực kì quan trọng, không nên phí toàn bộ viewport vào một hình hero (thực tế, hero này theo em nghĩ bỏ luôn cũng được)